packages/components/src/Inline/Inline.stories.tsx

```diff
@@ -38,6 +42,16 @@ const meta = {
       options: ['top', 'center', 'bottom'],
       description: 'Set the Vertical Alignment',
     },
+    dynamicAlign: {
+      control: {
+        type: 'boolean',
+      },
+      table: {
```

Comment:
use `args` prop for default value you can also put it just on the one story this feature is shown

Reply:
yes :)